src/app/routes/product-routes.js

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+import { products } from '../models/products';
+import { reviews } from '../models/reviews';
+
+const router = express.Router();
+
+router.get('/', function (request, response) {
```

Comment:
if you have the same routes you can use like `router.route('/').get(...).post(...)`